packages/react/types/react.d.ts

```diff
@@ -208,3 +208,6 @@ declare module '@lynx-js/types' {
 }
 
 export type { JSX } from '../runtime/jsx-runtime/index.d.ts';
+
+declare const _default: typeof import('@lynx-js/react');
```

Comment:
It seems like `@lynx-js/react` exactly points to `packages/react/types/react.d.ts`. It doesn't seem correct to use the types of itself.

Reply:
But this is not correct. We should not provide a wrong default export.